Tests/LibTLS/ca_certs.ini

```diff
@@ -0,0 +1,476 @@
+[Server CA 1B]
```

Comment:
Can this file be a symlink to the original instead?

Reply:
Done!